packages/sequelize-transparent-cache/src/cache/util.js

```diff
@@ -35,7 +35,7 @@ function loadAssociations (model) {
   Object.keys(model.associations).forEach((key) => {
     //  model.associations[key] does not work on include, we grab it from sequelize.model()
     if (model.associations[key].hasOwnProperty('options')) {
-      const modelName = model.associations[key].options.name.singular
+      const modelName = model.associations[key].target.name
       associations.push({
         model: model.sequelize.model(modelName),
```

Comment:
Even though this way of fetching the name is more 'secure', not having a check if the model is defined would risk throwing an unhandled error just like described on https://github.com/DanielHreben/sequelize-transparent-cache/issues/34